test/spec.test.ts

```diff
@@ -0,0 +1,81 @@
+import * as ts from "typescript";
+import yamljs from "yamljs";
+
+import { MetadataGenerator, SpecGenerator, Swagger } from "typescript-rest-swagger";
+
+import tsConfigBase from "../tsconfig.json";
+
+describe("test openapi spec against routes", () => {
+    test("all route names should match the generated ones", async () => {
+        const manualOpenApiSpec: Swagger.Spec = yamljs.load("src/modules/coreHttpApi/openapi.yml");
+
+        const files = "src/modules/**/*.ts";
+
+        const tsConfig = ts.convertCompilerOptionsFromJson(tsConfigBase.compilerOptions, process.cwd()).options;
+        const metadata = new MetadataGenerator([files], tsConfig).generate();
+        const defaultOptions = {
+            basePath: "/",
+            collectionFormat: "",
+            description: "",
+            entryFile: files,
+            host: "localhost:3000",
+            license: "",
+            name: "",
+            outputDirectory: "",
+            version: "",
+            yaml: false
+        };
+        const generator = new SpecGenerator(metadata, defaultOptions);
+        const generatedOpenApiSpec: Swagger.Spec = await generator.getOpenApiSpec();
+        harmonizeSpec(manualOpenApiSpec);
+        harmonizeSpec(generatedOpenApiSpec);
+
+        const manualPaths = Object.keys(manualOpenApiSpec.paths);
+        const generatedPaths = Object.keys(generatedOpenApiSpec.paths);
+
+        generatedPaths.forEach((path) => {
+            expect(manualPaths).toContain(path);
+        });
+
+        // Paths not defined in the typescript-rest way
+        const ignorePaths = ["/health", "/Healthcheck", "/Monitoring/Version", "/Monitoring/Requests", "/Monitoring/Support"];
```

Comment:
The difference between `manualPaths` and `generatedPaths` consists of `["/health", "/Monitoring/Version", "/Monitoring/Requests", "/Monitoring/Support"]`, but does not contain `"/Healthcheck"`.

In fact, neither `manualPaths` nor `generatedPaths` contain `"/Healthcheck"`. Just throwing this out there, in case this is unintentional.